src/apiCalls.js

```diff
@@ -0,0 +1,29 @@
+export const postData = (type, dataObject) => {
+  return fetch(`http://localhost:3001/api/v1/${type}`, {
+    method: 'POST',
+    body: JSON.stringify(dataObject),
+    headers: {
+      'Content-type': 'application/json'
+    }
+  })
+}
```

Comment:
I believe you need a error handler on the post.

Reply:
Thanks Hayley!  I have that in my scripts.js code after I make this fetch request. I believe the fetch API does not give us a bad response from the server in a .catch here: [](https://github.com/github/fetch/issues/155)